src/Api/Teams/Projects.php

```diff
@@ -34,10 +36,29 @@ public function list(array $params = [])
         return $this->get($path, $params);
     }
 
+    /**
+     * @throws Exception
+     *
+     * @return array
+     */
+    public function all()
```

Comment:
Please delete this method. Fetching all is already supported by the `ResultPager` class.